pkg/productinfo/alibaba/productinfo_alibaba.go

```diff
@@ -1,15 +1,43 @@
 package alibaba
 
 import (
-	"errors"
+	"encoding/json"
 	"fmt"
 	"github.com/aliyun/alibaba-cloud-sdk-go/sdk/log"
```

Comment:
please don't use the `aliyun` package to log messages, but `logrus` instead as everywhere else in the project